### PR TITLE
Upcoming movies screening

### PR DESCRIPTION
**Pull request by FelixHultman**
Added upcoming screenings to the website. 
logic for sorting out only upcoming screenings

screen

### CODE REVIEW

**Comment by FelixHultman** [comment]
> So far what i can see it looks good :) Except that sass needs to be moved, but thats a minor fix 👍🏼

should be fixed now, thanks!

File: package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
     "ejs": "^3.1.9",
     "express": "^4.18.2",
     "marked": "^11.2.0",
-    "node-fetch": "^3.3.2"
+    "node-fetch": "^3.3.2",
+    "sass": "^1.70.0"
```
[the remark this answers is not in the document]
Comment (FelixHultman) [reply package.json]: fixed!

**Review by MasakiShiraishi — APPROVED** [review]

**Review by JrSchoolAccount — APPROVED** [review]
👍

**Review by FelixHultman — COMMENTED** [review]
fixed now!

**Pull request merged**